full-service/src/json_rpc/v2/api/wallet.rs

```diff
@@ -605,9 +605,8 @@ where
                 });
             }
 
-            // let account_id: AccountId =
-            // account_id.as_str().try_into().map_err(format_error)?;
-            let account_id = AccountId::from([0u8; 32]);
+            // convert hex encoded string representation of account_id to AccountId struct
```

Comment:
nit: this feels like a narration comment. suggest removing it. 
Narration comments are comments that try to restate what the code is doing in conversational language. Generally there is more ambiguity in human language. If the code is doing some complex logic or optimizations it may make sense for a good explanatory comment, but this doesn't feel like one of those comments.


The `AccountId::` communicates the struct. The `try_from()` communicates that it's a conversion into the struct.
```suggestion
```

Reply:
agreed!